tests/sentry/api/endpoints/test_sentry_internal_app_tokens.py

```diff
@@ -106,5 +106,45 @@ def test_token_is_masked(self):
         response = self.client.get(url, format="json")
         response_content = json.loads(response.content)
 
+        assert len(response_content) == 1
         assert response_content[0]["token"] == MASKED_VALUE
         assert response_content[0]["refreshToken"] == MASKED_VALUE
+
+    def test_no_scope_api_token(self):
```

Comment:
Can we name this something like `test_api_token_cannot_escalate_scopes` or something